.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on pull requests to a set of branches.
+on:
+  pull_request:
+    branches: [
+      # This is our working branch
+      develop,
+      # This is where pull requests from "bors r+" are built.
+      staging,
+      # This is where pull requests from "bors try" are built. 
+      trying,
+    ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # no freebsd support for now =\
+        os: [macos-latest, windows-latest, ubuntu-latest]
```

Comment:
do we support windows builds?

Reply:
Yes - we have lots of people smeshing on windows